.github/workflows/docs-deployment.yml

```
@@ -0,0 +1,50 @@
+name: Docs Deployment
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: ${{ github.repository }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  build_docs:
+    name: Build docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: yarn
+      - uses: actions/configure-pages@v4
+      - run: yarn install
+      - name: Build docs site
+        run: |
+          yarn docs:build
+          touch docs/.vitepress/dist/.nojekyll
```

Seems that `.nojekyll` this is not needed anymore. 
It got removed from [github pages deploy docs](https://vitepress.dev/guide/deploy#github-pages) in march by https://github.com/vuejs/vitepress/pull/3630

```suggestion
        run: yarn docs:build
```

---

Great catch, thanks! I've made this change.